### PR TITLE
Remove ORC SSP 3 cert, it's expiring.

### PR DESCRIPTION
**Pull request by zachmargolis**
- We have its replacement ORC SSP 4 already (expires in 2024)

### CODE REVIEW

**Review by mitchellhenke — APPROVED** [review]
👍🏼

**Comment by zachmargolis** [comment]
(quick note, I'm going to wait until after Friday to merge this, I want to see the slack alert happen as expected, and then I'll merge this to resolve that)

**Pull request merged**
